README.md

```diff
@@ -0,0 +1,226 @@
+<!--
+    This Source Code Form is subject to the terms of the Mozilla Public
+    License, v. 2.0. If a copy of the MPL was not distributed with this
+    file, You can obtain one at http://mozilla.org/MPL/2.0/.
+-->
+
+<!--
+    Copyright 2020 Joyent, Inc.
+-->
+
+# Centos Image Builder
+
+This repo allows one to create custom CentOS images for use with Triton.
+
+Configuration files and scripts that are common to many images should be
+maintained in [sdc-vmtools](https://github.com/joyent/sdc-vmtools).
+
+## Requirements
+
+In order to use this repo, you need to have a SmartOS "joyent" brand zone that
+is capable of running qemu.  In order to run qemu the instance needs
+customization beyond what can be done with Triton APIs.  That is, an operator
+needs to customize the instance.  This is typically accomplished by running the
+following commands on the apprporiate compute node:
+
+```
+uuid=XXX	# Change this to the instance uuid
+
+topds=zones/$uuid/data
+zfs create -o zoned=on -o mountpoint=/data $topds
+
+zonecfg -z $uuid <<EOF
+add dataset
+set name=$topds
+end
+add fs
+set dir=/smartdc
+set special=/smartds
+set type=lofs
+set options=ro
+end
+add device
+set match=kvm
+end
+EOF
+```
+
+## Setup
+
+This relies on the sdc-vmtools repo as a submodule.  You can get the right
+version of that with:
+
+```
+git submodules update --init
+```
+
+If you forget to do that, `create-image` will do it before it tries to use
+anything from that submodule.
+
+## Using
+
+To generate a CentOS `<version>` image run:
+
+```
+# ./create-image -r <version>
+```
+
+While the primary focus of `create-image` is CentOS, it should be
+straight-forward to generate RHEL and Fedora images with this repo.  Once
```

Comment:
As I asked on the Debian one, should you cross-reference Debian image builds to that repo?

Reply:
Will link to https://github.com/joyent?q=mi--hvm